src/actions/auth.js

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios';
+
+import { registerConstants } from '../config/ActionTypes';
+import { BACKEND_URL } from '../config/constants';
+
+export function registerUser({ email, password }) {
```

Comment:
Can you try using only ES6 syntax and not mix the two styles?

Reply:
Done